src/libraries/Common/src/Interop/Unix/System.Native/Interop.GetCpuUtilization.cs

```diff
@@ -16,6 +16,6 @@ internal struct ProcessCpuInformation
         }
 
         [LibraryImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetCpuUtilization")]
-        internal static partial int GetCpuUtilization(ref ProcessCpuInformation previousCpuInfo);
+        internal static partial double GetCpuUtilization(ref ProcessCpuInformation previousCpuInfo);
```

Comment:
If you change this, you also need to change: https://github.com/dotnet/runtime/blob/7414af2a5f6d8d99efc27d3f5ef7a394e0b23c42/src/native/libs/System.Native/pal_time.c#L71